src/renderer/components/opds/BrowserResult.tsx

```diff
@@ -114,15 +124,17 @@ export class BrowserResult extends React.Component<BrowserResultProps, IState> {
         </div>;
     }
 
-    private browseOpds() {
-        const { url, location } = this.props;
+    private browseOpds(url: string) {
+        this.setState({browserResult: undefined});
```

Comment:
sorry to take the place to Daniel for reviewing. But is it compulsory to duplicate this line with https://github.com/readium/readium-desktop/pull/733/files#diff-e2c9e19293210a76adb1046d74cdbfd5R145

Reply:
I don't really understand, your link is pointing to the file differences but not a specific line

Reply:
look just above apiAction.

Reply:
This comment has been addressed: https://github.com/readium/readium-desktop/pull/733/commits/e5ff8aa507a6d402750fbe186fd264565f3c0ad1